ios/Classes/SwiftNewrelicMobilePlugin.swift

```diff
@@ -41,6 +41,7 @@ public class SwiftNewrelicMobilePlugin: NSObject, FlutterPlugin {
                 NewRelic.disableFeatures(NRMAFeatureFlags.NRFeatureFlag_WebViewInstrumentation)
             }
             if(args?["interactionTracingEnabled"] as! Bool == false) {
```

Comment:
Should NRFeatureFlag_DefaultInteractions and NRFeatureFlag_InteractionTracing be handled separately? I know we handle them separate in iOS.

In order to fully disable all interaction tracing including the default interactions a client can call 

```
                NewRelic.disableFeatures(NRMAFeatureFlags.NRFeatureFlag_InteractionTracing)
```



In order to just enable interaction tracing but disable default interactions this should be called. 

```
                NewRelic.disableFeatures(NRMAFeatureFlags.NRFeatureFlag_DefaultInteractions)
```

Reply:
not handling separately, on flutter side we only support custom interactions. this is same for all the hybrid platforms.